modules/rank-eval/src/test/java/org/elasticsearch/index/rankeval/RecallAtKTests.java

```diff
@@ -173,6 +173,7 @@ public void testXContentRoundtrip() throws IOException {
         }
     }
 
+    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/104569")
     public void testXContentParsingIsNotLenient() throws IOException {
```

Comment:
I think `RatedRequestsTests::testXContentParsingIsNotLenient` should been muted instead of this one

Reply:
@jan-elastic closed this because I did not quickly figure out where the leaked searches were. Search team probably needs to figure it out.